internal/cli/verify_install.go

```diff
@@ -0,0 +1,35 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/tilt-dev/tilt/internal/analytics"
+	engineanalytics "github.com/tilt-dev/tilt/internal/engine/analytics"
+)
+
+type verifyInstallCmd struct {
+}
+
+func (c *verifyInstallCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "verify-install",
+		Short: "Verifies Tilt Installation",
+	}
+	return cmd
+}
+
+func (c *verifyInstallCmd) run(ctx context.Context, args []string) error {
+	a := analytics.Get(ctx)
+	machineID, _ := a.GlobalTag("machine")
```

Comment:
you can remove this, it's unnecessary

Reply:
I'm sort of unclear on this...won't we need lines 24-25 to get the "machine" tag from analytics?

Reply:
nope! it's all handled for you by the framework. If you look at other calls throughout tilt, none of them do this, and they all have the machine tag attached.

Reply:
Hmmm I'm still sort of confused....
Wouldn't we need `cmdVerifyInstallTags` to do `a.Incr()`?